package/src/components/MessageList/hooks/useMessageList.ts

```diff
@@ -48,10 +52,11 @@ export const useMessageList = <
 >(
   params: UseMessageListParams,
 ) => {
-  const { deletedMessagesVisibilityType, inverted, noGroupByUser, threadList } = params;
```

Comment:
Any specific reason for consuming `deletedMessagesVisibilityType` from context instead of params? And if we do change it, we need to remove this param from the usage of `useMessageList` hook, no?

Reply:
Passing it through as a param here seemed redundant since we already have it in the Messages context - am I missing some original intention on why it's better to pass it through as a parameter?

> we need to remove this param from the usage of useMessageList hook, no?

I'm not sure I get what you mean, afaik it's already removed from the param list?

Reply:
I mean here, in `MessageList.tsx`
<img width="605" alt="Screenshot 2022-09-22 at 11 28 21" src="https://user-images.githubusercontent.com/11586388/191711303-1f6a587b-bd56-4ded-bcef-fe02a77475fe.png">

Reply:
Wonder how typescript didn't catch this. Removed this, and also all other references to `deletedMessagesVisibilityType` from `MessageList` since it's not being used there